public/components/model/model_versions_panel/model_version_status_detail.tsx

```diff
@@ -161,10 +166,15 @@ export const ModelVersionStatusDetail = ({
               </Link>
             )}
           </EuiText>
-          <EuiSpacer size="s" />
-          <EuiText>
-            <b>{timeTitle}:</b> {renderTime(restProps[timeField], DATE_FORMAT)}
-          </EuiText>
+          {timeTitle && (
+            <>
+              <EuiSpacer size="s" />
+              <EuiText>
+                <b>{timeTitle}:</b>{' '}
+                {timeValue && dateFormat ? renderTime(timeValue, dateFormat) : '-'}
```

Comment:
It looks better to me if we do:
```javascript
{timeValue ? renderTime(timeValue, dateFormat) : '-'}
```
This `dateFormat` can fallback to a default value

Reply:
Is that mean use the constant date format as fallback?

Reply:
@wanglam I'd suggest all places which conditionally render the date&time based on `dateFormat` should be changed to use a fallback strategy.

Reply:
Sure. I am working on create an separate time display component to handle these.